apps/src/utils.js

```diff
@@ -559,14 +568,45 @@ export function normalize(vector) {
   };
 }
 
+type Position =
+    1
+  | 2
+  | 3
+  | 4
+  | 5
+  | 6
+  | 7
+  | 8
+  | 9
+  | 10
+  | 11
+  | 12
+  | 13
+  | 14
+  | 15
+  | 16
+  | 17
+  | 18
+  | 19
+  | 20
+  | 21
+  | 22
+  | 23
+  | 24
+  | 25;
```

Comment:
yeah. We can change constants.Position to look more like an enum, and then declare the type based on the keys in the "enum object".